ome_zarr/cli.py

```diff
@@ -73,7 +73,7 @@ def csv_to_labels(args: argparse.Namespace) -> None:
     csv_to_zarr(args.csv_path, args.csv_id, args.csv_keys, args.zarr_path, args.zarr_id)
 
 
-def main(args: List[str]) -> None:
+def main(args: Union[List[str], None] = None) -> None:
```

Comment:
TIL: https://stackoverflow.com/questions/51710037/how-should-i-use-the-optional-type-hint#:~:text=There%20is%20technically%20no%20difference,are%20exactly%20the%20same%20thing.

Looking forward to the new recommended "List[str] | None" >= Py 3.10

Reply:
oh right, Optional would've been nicer - it's Monday - sorry :). And I too look forward to the new syntax :)

Reply:
It's Monday indeed. You can always update in your next PR ;)

0.7.1 pushed and building.

Reply:
thank you :)

Reply:
No, no, no. Thank you! :)